client/src/components/App/index.tsx

```diff
@@ -41,10 +42,15 @@ export const AppComponent: FunctionComponent<AppProps> = observer(props => {
   const { api } = props;
   const [flowsDiffCount, setFlowsDiffCount] = useState({ value: 0 });
   const [eventStream, setEventStream] = useState<IEventStream | null>(null);
+  const [isStreaming, setIsStreaming] = useState<boolean>(true);
 
   const store = useStore();
   const notifier = useNotifier();
 
+  useEffect(() => {
+    setIsStreaming(Boolean(eventStream));
```

Comment:
Consider using shorter form instead: `!!eventStream`

Reply:
I don't like it :)